wdfn-server/waterdata/services/ogc.py

```diff
@@ -0,0 +1,35 @@
+"""
+Helpers to retrieve OGC data - Networks.
+"""
+from waterdata import app
+from waterdata.utils import execute_get_request
+
+
+def get_networks(network_cd):
+    """
+    Gets the network data from a json file
+    :param network_cd: collections-id
+    """
+    # Handle feature flag for network pages
+    if not app.config['NETWORK_PAGES_ENABLED']:
```

Comment:
That's fine if you don't want a feature flag here, I was just following the get_cooperators() pattern in services/sifta.py here, seems to have one there.  Should we not have a flag?

Reply:
That one is a little different in that it gets cooperator information either from a service (if enabled) or a static file. In our case, we feature flag the view which you are doing. Now that I double check the JIRA I see that a feature flag is not part of the acceptance criteria. You should verify with @jkreft-usgs  whether one is needed

Reply:
We can probably go without a feature flag on this one.  Greg will have to do an apache mapping before it can be publicly mapped on waterdata.usgs.gov anyways.